recipe/build.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# TELEMAC home directory
+export HOMETEL=$SRC_DIR/telemac-mascaret
+# Configuration file
+export SYSTELCFG=$HOMETEL/configs/systel.cfg
+
+# Configure PATH and PYTHONPATH
+export PATH=$HOMETEL/scripts/python3:$PATH
+export PYTHONPATH=$HOMETEL/scripts/python3 #:$PYTHONPATH
+
+rm -rf $HOMETEL/configs/*
+export MUMPSHOME=$PREFIX
+export SCALAPACKHOME=$PREFIX
+export BLACSHOME=$PREFIX
+export METISHOME=$PREFIX/lib
+
+#linux
+if [[ $(uname) == Linux ]]; then
+   cp $SRC_DIR/systel.linux.cfg $HOMETEL/configs/systel.cfg
+   export USETELCFG=gnu.dynamic.debug # generate also for debug mode
+   compile_telemac.py
+   export USETELCFG=gnu.dynamic
```

Comment:
```suggestion
   cp $SRC_DIR/systel.linux.cfg $HOMETEL/configs/systel.cfg
```

Removing compile_telemac.py
Not exporting USETELCFG and calling compile_telemac will compile and configuration (like it's done for windows)

Reply:
I know that by default it compiles all configs present in the syste.cfg
problem is it doesn't here .. I don't know why
this was my fix

Reply:
I'll use your suggestion for now but will package using version, to produce both builds 
otherwise it won't work. 

I will add this as an issue to fix for later